src/components/ChartMana/index.js

```diff
@@ -19,14 +19,17 @@ export default React.memo(function ChartMana(props) {
   const [faction, setFaction] = React.useState('*')
   const [level, setLevel] = React.useState(5)
 
+  const activeFaction = faction.split(',')
+  const mainFaction = [...activeFaction].pop()
```

Comment:
This might be a little safer because it does not rely on order:
```suggestion
  const factions = faction.split(',')
  const mainFaction = factions.filter(faction => faction !== 'neutral')
```

Reply:
Won't this cause an issue with the 'neutral' faction?

Reply:
Ah yes. You’re right. Then maybe we check the length before filtering?
```js
const mainFaction = factions.length === 1 ? factions[0] : factions.filter(faction => faction !== 'neutral')
```